tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -849,6 +849,48 @@ public function testAddIndexWithLimit()
         $this->assertEquals($expected_limit, 50);
     }
 
+    public function testAddMultiIndexesWithLimitSpecifier()
+    {
+        $table = new \Phinx\Db\Table('table1', [], $this->adapter);
+        $table->addColumn('email', 'string')
+              ->addColumn('username', 'string')
+              ->save();
+        $this->assertFalse($table->hasIndex(['email', 'username']));
+        $table->addIndex(['email', 'username'], ['limit' => [ 'email' => 3, 'username' => 2 ]])
+              ->save();
+        $this->assertTrue($table->hasIndex(['email', 'username']));
+        $index_data = $this->adapter->query(sprintf(
+            'SELECT SUB_PART FROM information_schema.STATISTICS WHERE TABLE_SCHEMA = "%s" AND TABLE_NAME = "table1" AND INDEX_NAME = "email" AND COLUMN_NAME = "email"',
+            TESTS_PHINX_DB_ADAPTER_MYSQL_DATABASE
+        ))->fetch(\PDO::FETCH_ASSOC);
+        $expected_limit = $index_data['SUB_PART'];
+        $this->assertEquals($expected_limit, 3);
+        $index_data = $this->adapter->query(sprintf(
+            'SELECT SUB_PART FROM information_schema.STATISTICS WHERE TABLE_SCHEMA = "%s" AND TABLE_NAME = "table1" AND INDEX_NAME = "email" AND COLUMN_NAME = "username"',
+            TESTS_PHINX_DB_ADAPTER_MYSQL_DATABASE
+        ))->fetch(\PDO::FETCH_ASSOC);
+        $expected_limit = $index_data['SUB_PART'];
+        $this->assertEquals($expected_limit, 2);
+    }
+
+    public function testAddSingleIndexesWithLimitSpecifier()
+    {
+        $table = new \Phinx\Db\Table('table1', [], $this->adapter);
+        $table->addColumn('email', 'string')
+            ->addColumn('username', 'string')
+            ->save();
+        $this->assertFalse($table->hasIndex('email'));
+        $table->addIndex('email', ['limit' => [ 'email' => 3, 2 ]])
```

Comment:
Is the number 2 at the end used for anything?

Reply:
No, just making sure it was running normally. When used like that, number 2 should be ignored.